wdl-analysis/src/stdlib.rs

```diff
@@ -1581,6 +1610,16 @@ pub static STDLIB: LazyLock<StandardLibrary> = LazyLock::new(|| {
                             .parameter(PrimitiveTypeKind::String)
                             .ret(PrimitiveTypeKind::String)
                             .build(),
+                        // This overload isn't explicitly specified in the spec, but the spec
```

Comment:
Should we propose updates to the spec for these? Or no?

Reply:
It's _mentioned_ in the spec in the same section, but there probably should be explicit overloads for them in the list of signatures. I'll file a spec issue.

Reply:
Opened https://github.com/openwdl/wdl/issues/683.